logger.c

```diff
@@ -689,6 +728,39 @@ static void _logger_log_item_store(logentry *e, const enum store_item_type statu
     e->size = sizeof(struct logentry_item_store) + nkey;
 }
 
+static void _logger_log_conn_event(logentry *e, struct sockaddr *addr,
+        enum network_transport transport, enum close_reasons reason, int sfd) {
+    struct logentry_conn_event *le = (struct logentry_conn_event *) e->data;
+
+    memset(le->rip, 0, sizeof(le->rip));
+    le->sfd = sfd;
+
+    switch (addr->sa_family) {
```

Comment:
Actually noodling this a bit; it's most optimal to do the binary->text conversion in the logger thread (parse-ise) and do a fast memcpy of the sockaddr stuff here? Any reason not to?

In general the inline log events should be as lean as possible and then we backload all parsing/decoding into the logger thread.

Reply:
Makes sense. I pushed up 8579cadf1258d62864971672fe3fcb2c34b5fe8d which moves this block of logic into the logger thread and keeps the hot path minimal. Let me know what you think (it's a bit more verbose).